react/features/remote-video-menu/actions.js

```diff
@@ -12,3 +27,56 @@ import { RemoteVideoMenu } from './components';
 export function hideRemoteVideoMenu() {
     return hideDialog(RemoteVideoMenu);
 }
+
+/**
+ * Mutes the local participant.
+ *
+ * @returns {Function}
+ */
+export function muteLocal() {
```

Comment:
Since this is very similar to what we do in AudioMuteButton.js, I'm wondering can't we start dispatching it there too?

Reply:
Updated AudioMuteButton.js to this new action!